Modules/TRD/src/TrackletPerTriggerCheck.cxx

```diff
@@ -102,10 +102,14 @@ Quality TrackletPerTriggerCheck::check(std::map<std::string, std::shared_ptr<Mon
 
       // applying check
       float MeanTracletPertrigger = h->GetMean();
-      if (MeanTracletPertrigger > mDesiredMeanRegion.second && MeanTracletPertrigger < mDesiredMeanRegion.first) {
+      if (MeanTracletPertrigger < mDesiredMeanRegion.second && MeanTracletPertrigger > mDesiredMeanRegion.first) {
```

Comment:
to avoid these kind of typos wouldn't it be better to use two variables in your checker class, e.g. `mMeanRegionLow` and `mMeanRegionHigh` or something like that? And then for reading the parameters from json file I'd prefer you use the common `getFromConfig` as it is done for example in https://github.com/AliceO2Group/QualityControl/blob/a0dc3f4234734b36d0b3dc9ce54cb3a3d305e591/Modules/TRD/src/DigitsTask.cxx#L408-L415. You'll need to include `Common/Utils.h` for this

Reply:
I updated this PR with the above suggestions.